src/libsyncengine/jobs/network/networkjobsparams.h

```diff
@@ -67,6 +67,21 @@ static const std::string cursorKey = "cursor";
 static const std::string hasMoreKey = "has_more";
 static const std::string tokenKey = "token";
 static const std::string preferenceKey = "preference";
+static const std::string applicationKey = "application";
```

Comment:
I am wondering whether these definitions of keys, or even just their litteral values, should into the body of `GetAppVersionJob::toDistributionChannel`?